app/src/components/features/rules/RulesListContainer/RulesTable/index.js

```diff
@@ -243,6 +247,13 @@ const RulesTable = ({
     dispatch(actions.updateRulesToPopulate(rules));
   };
 
+  const getPrettyDesktopRuleTooltipTitle = (ruleType) => {
+    if (ruleType === GLOBAL_CONSTANTS.RULE_TYPES.REDIRECT) {
```

Comment:
`isDesktopOnlyRule()` already checks for rule type to be redirect. Is this check necessary?

Reply:
`isDesktopOnlyRule()` will also have a check for `GraphQL` resource type in response rule, thats why the tooltip content is handled separately